library/src/main/kotlin/build/buf/connect/protocols/GRPCInterceptor.kt

```diff
@@ -188,11 +188,9 @@ internal class GRPCInterceptor(
 
     private fun Headers.withGRPCRequestHeaders(): Headers {
         val headers = toMutableMap()
-        // Note that we do not comply with the recommended structure for user-agent:
-        // https://github.com/grpc/grpc/blob/v1.51.1/doc/PROTOCOL-HTTP2.md#user-agents
-        // But this behavior matches connect-web:
-        // https://github.com/bufbuild/connect-web/blob/v0.4.0/packages/connect-core/src/grpc-web-create-request-header.ts#L33-L36
-        headers[GRPC_USER_AGENT] = listOf("@bufbuild/connect-kotlin")
```

Comment:
This header was wrong - gRPC uses the standard `User-Agent` header, only gRPC-web uses a different one.